README.md

```diff
@@ -224,6 +224,104 @@ messages with an unrecognised conversation ID on a dead letter queue.
+2. Polling the request: after receiving a 202 response, we recommend polling the endpoint at regular intervals using an increasing call gap strategy. Each poll can return the following responses:-
+   1. 204 No content: this response indicates that we are still processing the requests / waiting for the EHR message response.
+   2. 200 Success: this response indicates we have successfully received and converted the EHR to JSON; you will also receive the FHIR bundle in the response's body.
+   3. 400,404,500,501: The endpoint can return all these possible error codes. These will all provide a detailed error with an operationOutcome JSON model response in the body. This looks like...
+ 
+   {
+      "resourceType": "OperationOutcome",
+      "meta": {
+         "profile": [
+            "https://fhir.nhs.uk/STU3/StructureDefinition/GPConnect-OperationOutcome-1"
+         ]
+      },
+      "issue": [
+         {
+            "severity": "error",
+            "code": "exception",
+            "details": {
+               "coding": [
+                  {
+                     "system": "https://fhir.nhs.uk/STU3/ValueSet/Spine-ErrorOrWarningCode-1",
+                     "code": "PATIENT_NOT_FOUND",
+                     "display": "GP2GP - Patient is not registered at the practice"
+                  }
+               ]
+            }
+         }
+      ]
+   }
+
+### /$gpc.ack
+
+The ack endpoint is the final endpoint to call once you are happy that the EHR record you have received is acceptable.
+If you do not call this endpoint after receiving an EHR from the migratestructuredrecord enpoint, then you risk the losing practise triggering off the manual postal transfer.
+
+The following is required to call this endpoint:-
+
+- CONVERSATION_ID: The id associated with the patient transfer request.
+- CONFIRMATION_RESPONSE: you can provide the following status:-
+   - ACCEPTED: This will tell the sending incumbent that you are happy with the received EHR.
+   - ABA_INCORRECT_PATIENT: Alert the sender that this is not the patient you expected.
+   - NON_ABA_INCORRECT_PATIENT: Alert the sender that this is not the patient you expected.
+   - FAILED_TO_INTEGRATE: You have encountered a problem integrating the record into your system. This will alert the sender to an error and trigger off the postal process.
+   - SUPPRESSED:
+
+Endpoint calling:
+
+This endpoint is a fire-and-forget endpoint.
+If your request is successful, you will get a 200: Success response.
+If your request is unsuccessful, you will get a 500: Server error response.
+If you receive a 500 response, you can retry again at any point, however, it should be noted that you must receive a 200: Success response from the migratestructuredrecord for the given conversation ID to receive a 200: Success from this endpoint.
+
+Note: To improve reliability on this endpoint we are currently looking at a polling change, the documentation will be updated once this has been updated.
```

Comment:
Is this the change I mentioned in the previous comment? If so, ignore it.

Reply:
Afraid not, this change is in regards to changing the ack endpoint from a fire and forget to a polling endpoint.